src/ocm/components/clusterWizard/ClusterPlatformIntegrationHint.tsx

```diff
@@ -12,16 +12,21 @@ export const ClusterPlatformIntegrationHint: React.FC<ClusterPlatformIntegration
   platformType,
 }) => {
   const { isPlatformIntegrationSupported } = useClusterSupportedPlatforms(clusterId);
-  if (isPlatformIntegrationSupported && platformType !== 'vsphere') {
+  if (isPlatformIntegrationSupported) {
     return (
       <Alert
-        title="Discover the full potential of vSphere integration"
+        title={
+          platformType === 'vsphere'
```

Comment:
Again, `platformType` vsphere represents an already active platform integration.
You should be checking which platform integration is enabled.
(See a separate comment about `useClusterSupportedPlatforms`)